tasks/install/Debian.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Debian - Ensure haproxy apt repo is added.
   apt_repository:
-    repo: "deb {{ haproxy_repo }}"
+    repo: "deb {{ haproxy_repo }} {{ ansible_distribution_release }} main"
```

Comment:
This is a revert of f15e74b. Is this really what you want to do?

Reply:
Sorry I didn't apply the rebase correctly in this branch